ppx/browser/dune

```diff
@@ -0,0 +1,48 @@
+(library
+ (public_name melange-json.browser)
+ (name ppx_deriving_json_browser)
+ (modules
+  :standard
+  \
+  ppx_deriving_json_runtime
+  ppx_deriving_json_browser_test)
+ (libraries ppxlib containers)
```

Comment:
could we remove the `containers` dependency? What's the biggest argument in favor of keeping it?

Reply:
I did this in https://github.com/melange-community/melange-json/pull/10/commits/b5aa8a5cab2c6de96e36f7241f1b6c856929e114